cockatrice/src/filtertreemodel.cpp

```diff
@@ -4,6 +4,7 @@
 #include "filtertree.h"
 
 #include <QFont>
+#include <qnamespace.h>
```

Comment:
That looks a private header, maybe `#include <Qt>` will work?

Reply:
ah, yeah, that seems to have been autogenerated.